core/trino-main/src/test/java/io/trino/sql/TestExpressionInterpreter.java

```diff
@@ -1397,6 +1421,61 @@ public void testLikeOptimization()
         assertOptimizedEquals("unbound_string LIKE unbound_pattern ESCAPE unbound_string", "unbound_string LIKE unbound_pattern ESCAPE unbound_string");
     }
 
+    @Test
+    public void testLikeCharOptimization()
+    {
+        // constant literal pattern of length shorter than value length
```

Comment:
nit:
`patter of length smaller than` or `pattern shorter than`

Reply:
not changing then